packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -154,6 +154,15 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
     }
   }
 
+  public getStartItemIndexInView(): number {
+    if (this._list && this._list.current) {
+      return this._list.current.getStartItemIndexInView() || 0;
+    } else if (this._groupedList && this._groupedList.current) {
+      return this._groupedList.current!.getStartItemIndexInView() || 0;
```

Comment:
if you have the null check, we should not need the '!' after this._groupedList.current